packages/desktop-client/src/components/filters/FiltersMenu.js

```diff
@@ -181,7 +181,8 @@ function ConfigureField({
         <View style={{ marginBottom: 10 }}>
           <Stack direction="row" align="flex-start">
```

Comment:
And also I think we can remove the `Stack` wrapping component and the `<View style={{ flex: 1 }} />`.

Reply:
Thank you for your feedback, are you suggesting to remove both <View> and <Stack> components?

Reply:
Just the `View` component can be removed and the `style` prop on `Select`. I think that would solve the problem (but please double check).

Thanks again!